setup.py

```diff
@@ -16,14 +15,14 @@ def read(fname):
 setup(
     name="authzerolib",
     py_modules=['authzero'],
-    version="1.0.6",
-    author="Guillaume Destuynder",
-    author_email="gdestuynder@mozilla.com",
+    version="1.1.0",
+    author="Mozilla IAM",
+    author_email="gene@mozilla.com",
     description=("A super simple and basic client lib for Auth0"),
-    license="MPL",
+    license=" MPL-2.0",
```

Comment:
There's extra whitespace here that may cause trouble for packaging sites?

Reply:
Whoops, good catch.